source/user-manual/reference/ossec-conf/email_alerts.rst

```diff
@@ -1,5 +1,8 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
+.. meta::
+  :description: Check out more about the local configuration of Wazuh. You can learn more about the email options configured in the <global> section here. 
```

Comment:
I'd rewrite this meta. The second sentence is not descriptive enough and the meaning is not clear.